packages/devtools_app/test/integration_tests/integration.dart

```diff
@@ -246,11 +245,11 @@ class AppResponse {
 class AppError {
   AppError(this.json);
 
-  final Map<dynamic, dynamic> json;
+  final Map<dynamic, dynamic>? json;
```

Comment:
make non nullable if we are null asserting unconditionally elsewhere

Reply:
done